### PR TITLE
Explain how to sync an existing calendar

### PR DESCRIPTION
**Pull request by clochix**
A contribution from our user Aleth Gueguen

### CODE REVIEW

File: src/documents/en/mobile/calendar.html.md [hunk 1]
```diff
@@ -247,6 +247,27 @@ Congratulations, your computer is now synchronized with your Cozy!
 That's it! You can seamlessly manage your events from your computer and your Cozy.
 
 
+
+<br />
+<br />
+
+#### How to synchronize an existing calendar
+
+If you want to synchronize an existing calendar with your Cozy, you will have to perform some manipulations. Create a new calendar inside Cozy, import your existing calendar into it, then create a new local calendar synchronized with Cozy. Here we go :
```
Comment (frankrousseau) [inline src/documents/en/mobile/calendar.html.md]: There is no space before `:` in English.

File: src/documents/fr/mobile/calendar.html.md [hunk 2]
```diff
@@ -249,6 +249,29 @@ Félicitations, votre ordinateur est maintenant synchronisé avec votre Cozy !
...
 
+<br />
+<br />
+
+
+#### Comment synchroniser un agenda existant
+
+Synchroniser un agenda déjà créé dans Thunderbird avec votre serveur Cozy demande quelques manipulations. Il faut créer un nouvel agenda dans Cozy, y importer votre agenda local, puis créer un nouvel agenda local synchronisé avec le serveur.
+
+Voici comment faire depuis Thunderbird
```
Comment (frankrousseau) [inline src/documents/fr/mobile/calendar.html.md]: Punctuation is missing at the end of this sentence.

**Comment by clochix** [comment]
Fixed ☺

**Comment by clochix** [comment]
I added a small note explaining how to re-generate the server certificate when using .deb package.

**Pull request merged**